integration-tests/api/medusa-config.js

```diff
@@ -17,11 +17,7 @@ module.exports = {
   modules: {
     cacheService: {
       resolve: "@medusajs/cache-inmemory",
-      // don't set cache since this is shared between tests
-      // and since we have "test-product" / "test-variant" as ids
-      // in a bunch of tests, this could cause that incorrect data is returned
-      // (e.g. price selection caches calculations under `ps:${variantId}`)
-      options: { ttl: 0 },
+      options: { ttl: 15 },
```

Comment:
suggestion: We can use process.env.XXX and set that when we create the server in the test scenario wdyt? That way it does not impact all the scenarios